lib/venue.rb

```diff
@@ -1,2 +1,31 @@
 class Venue
+  attr_accessor :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(name)
+    @patrons << name
+  end
+
+  def yell_at_patrons
+    uppercased_names = []
+    @patrons.each do |name|
+      uppercased_names << name.upcase
+    end
+    uppercased_names
+  end
+
+  def over_capacity?
+    @patrons.length > @capacity ? true : false
```

Comment:
Nice use of a ternary here. It will also work if you get rid of everything after `@capacity`. Since you are using the `>` operator it will return a boolean based on the condition.